docs/core/config/auth/databases/ldap.md

```diff
@@ -917,38 +907,31 @@ Create a "quota_rule" field with value `*:bytes=<n>` where `<n>` comes
 from "quotaBytes" LDAP attribute:
 
 ```
-user_attrs = =quota_rule=\*:bytes=%{ldap:quotaBytes}
+fields {
+  quota_rule = *:bytes=%{ldap:quotaBytes}
+}
 ```
 
-Create a "mail_path" field with value `/var/mail/<dir>/Maildir` where
+Create a "mail" field with value `maildir:/var/mail/<dir>/Maildir` where
 `<dir>` comes from "sAMAccountName" LDAP attribute:
 
 ```
-user_attrs = =mail_path=/var/spool/vmail/%{ldap:sAMAccountName}/Maildir
+fields {
+  mail = maildir:/var/spool/vmail/%{ldap:sAMAccountName}/Maildir
+}
```

Comment:
You're changing`mail_path` -> `mail` in here back to the old syntax. Should stay as mail_path without maildir: prefix.